internal/auth0/connection/expand.go

```diff
@@ -1078,3 +1078,40 @@ func checkForUnmanagedConfigurationSecrets(configFromTF, configFromAPI map[strin
 
 	return warnings
 }
+
+func expandSCIMConfigurationMapping(data *schema.ResourceData) *[]management.SCIMConfigurationMapping {
+	mapping := []management.SCIMConfigurationMapping{}
+	srcMapping := data.Get("mapping").(*schema.Set)
```

Comment:
Could we use the rawConfig instead? That's preferred in most cases.

Reply:
Using the RawConfig is generally not the best choice from Terraform's point of view, generally I prefer to use their schema abstraction except in cases (like detecting missing attributes) where that does not work